addons/mrp_account/tests/test_bom_price.py

```diff
@@ -111,15 +111,24 @@ def test_00_compute_price(self):
 
     def test_01_compute_price_operation_cost(self):
         """Test calcuation of bom cost with operations."""
-        workcenter_from1 = Form(self.env['mrp.workcenter'])
-        workcenter_from1.name = 'Workcenter'
-        workcenter_from1.time_efficiency = 80
-        workcenter_from1.default_capacity = 2
-        workcenter_from1.oee_target = 100
-        workcenter_from1.time_start = 15
-        workcenter_from1.time_stop = 15
-        workcenter_from1.costs_hour = 100
-        workcenter_1 = workcenter_from1.save()
+        workcenter_form1 = Form(self.env['mrp.workcenter'])
+        workcenter_form1.name = 'Workcenter'
+        workcenter_form1.time_efficiency = 80
+        workcenter_form1.default_capacity = 2
+        workcenter_form1.oee_target = 100
+        workcenter_form1.time_start = 15
+        workcenter_form1.time_stop = 15
+        workcenter_form1.costs_hour = 100
+        workcenter_1 = workcenter_form1.save()
+
+        capacity_dining_table = self.env['mrp.workcenter.capacity'].create({
+            'product_id': self.dining_table.id,
+            'workcenter_id': workcenter_1.id,
+            'time_start': 2,
+            'time_stop': 1,
+        })
+
+        workcenter_1.write({'capacity_ids': [(4, capacity_dining_table.id)]})
```

Comment:
This line:

`workcenter_1.write({'capacity_ids': [(4, capacity_dining_table.id)]})`

is redundant with the `'workcenter_id': workcenter_1.id,` in the `create`